spec/yaml/swiftnav/sbp/piksi.yaml

```diff
@@ -197,24 +173,32 @@ definitions:
  - MSG_UART_STATE:
     id: 0x0018
     public: True
-    desc: State of the UART channels.
+    short_desc: State of the UART channels.
+    desc: |
+      This message reports data latency and throughput of the UART
+      channels providing SBP I/O.
     fields:
         - uart_a:
             type: UARTChannel
-            desc: State of UART A.
+            desc: State of UART A (telemetry radio)
```

Comment:
These need not be telemetry radios.  They could be host access ports for embedded hosts, or Bluetooth or other interfaces in future.